types/react-jss/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
```

Comment:
@RyanCavanaugh I think this is ok since the package.json is not actually used for publishing. React and redux-form also have private set to true.

If I set private to false dtslint throws because it expects private to equal true.